### ISSUE TITLE
"Process failed to execute" when building dev kit

### ISSUE THREAD

**Issue by illus-coder** [CosmosOS/Cosmos#1623]
**Describe the problem**
When trying to build/install the dev kit, it throws an error and I'm not sure what I did wrong?

**Steps of installation up to now**
Run Install-VS2019 as admin/ or run it without admin, neither one helps

**Expected behavior**
Expecting the dev kit to install

**Logs**

========================================
    Builder Log
========================================

========================================
    Checking Dependencies
========================================

Repos: IL2CPU, XSharp and Common is installed.
Visual Studio 15.9+ is installed.
Inno Setup not found.

========================================
    Restore - Build.sln
========================================

"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\MSBuild.exe" "C:\Users\illus\Documents\Cosmos-master\Build.sln" /nologo /maxcpucount /nodeReuse:False /verbosity:minimal /t:"Restore" /p:DeployExtension=False
  Determining projects to restore...
  Skipping project "C:\Users\illus\Documents\IL2CPU\..\Cosmos\source\Cosmos.Debug.Kernel\Cosmos.Debug.Kernel.csproj" because it was not found.
  Skipping project "C:\Users\illus\Documents\IL2CPU\..\Cosmos\source\Cosmos.Debug.Kernel\Cosmos.Debug.Kernel.csproj" because it was not found.
C:\Program Files\dotnet\sdk\5.0.101\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [C:\Users\illus\Documents\IL2CPU\source\IL2CPU\IL2CPU.csproj]
C:\Program Files\dotnet\sdk\5.0.101\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [C:\Users\illus\Documents\Cosmos-master\Tools\NASM\NASM.csproj]
C:\Program Files\dotnet\sdk\5.0.101\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [C:\Users\illus\Documents\Cosmos-master\source\TheRingMaster\TheRingMaster.csproj]
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\Cosmos.Core.csproj (in 2.63 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Common\Cosmos.Common.csproj (in 2.63 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Build.Tasks\Cosmos.Build.Tasks.csproj (in 59.6 sec).
  Restored C:\Users\illus\Documents\Common\source\Cosmos.Build.Common\Cosmos.Build.Common.csproj (in 2.79 sec).
  Restored C:\Users\illus\Documents\XSharp\source\XSharp\XSharp\XSharp.csproj (in 594 ms).
  Restored C:\Users\illus\Documents\XSharp\source\Spruce\Spruce.csproj (in 3 ms).
  Restored C:\Users\illus\Documents\Common\source\VSPropertyPages\VSPropertyPages.csproj (in 1.73 min).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core_Plugs\Cosmos.Core_Plugs.csproj (in 1.87 min).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core_Asm\Cosmos.Core_Asm.csproj (in 1.87 min).
  Restored C:\Users\illus\Documents\Cosmos-master\Tests\Cosmos.TestRunner.TestController\Cosmos.TestRunner.TestController.csproj (in 8 ms).
  Restored C:\Users\illus\Documents\IL2CPU\source\IL2CPU.API\IL2CPU.API.csproj (in 7 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Deploy.USB\Cosmos.Deploy.USB.csproj (in 56 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Deploy.Pixie\Cosmos.Deploy.Pixie.csproj (in 32 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.Kernel\Cosmos.Debug.Kernel.csproj (in 5 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.Kernel.Plugs.Asm\Cosmos.Debug.Kernel.Plugs.Asm.csproj (in 7 ms).
  Restored C:\Users\illus\Documents\IL2CPU\source\IL2CPU.Debug.Symbols\IL2CPU.Debug.Symbols.csproj (in 6.85 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.GDB\Cosmos.Debug.GDB.csproj (in 17 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.HyperVServer\Cosmos.Debug.HyperVServer.csproj (in 8.14 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.DebugConnectors\Cosmos.Debug.DebugConnectors.csproj (in 7.87 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.Common\Cosmos.Debug.Common.csproj (in 759 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.Hosts\Cosmos.Debug.Hosts.csproj (in 9.25 sec).
  Restored C:\Users\illus\Documents\IL2CPU\source\Cosmos.IL2CPU\Cosmos.IL2CPU.csproj (in 80 ms).
  Restored C:\Users\illus\Documents\IL2CPU\source\Cosmos.Core.DebugStub\Cosmos.Core.DebugStub.csproj (in 11 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\Tools\NASM\NASM.csproj (in 14.45 sec).
  Restored C:\Users\illus\Documents\IL2CPU\source\IL2CPU\IL2CPU.csproj (in 55.36 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\TheRingMaster\TheRingMaster.csproj (in 19.63 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2_Plugs\Cosmos.System2_Plugs.csproj (in 13 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\Cosmos.System2.csproj (in 11 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\Cosmos.HAL2.csproj (in 12 ms).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.ProjectSystem\Cosmos.VS.ProjectSystem.csproj (in 16.52 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\Cosmos.VS.Windows.csproj (in 30.24 sec).
  Restored C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.DebugEngine\Cosmos.VS.DebugEngine.csproj (in 16.58 sec).

========================================
    Build - Build.sln
========================================

"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\MSBuild.exe" "C:\Users\illus\Documents\Cosmos-master\Build.sln" /nologo /maxcpucount /nodeReuse:False /verbosity:minimal /t:"Build" /p:"OutputPath=C:\Users\illus\Documents\Cosmos-master\Build\VSIP\\" /p:"VsixOutputPath=C:\Users\illus\Documents\Cosmos-master\Build\VSIP\\" /p:DeployExtension=False
C:\Program Files\dotnet\sdk\5.0.101\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [C:\Users\illus\Documents\IL2CPU\source\IL2CPU\IL2CPU.csproj]
  Spruce -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Spruce.dll
C:\Users\illus\Documents\Common\source\Cosmos.Build.Common\BochsSupport.cs(76,30): warning CS0168: The variable 'ex' is declared but never used [C:\Users\illus\Documents\Common\source\Cosmos.Build.Common\Cosmos.Build.Common.csproj]
  Cosmos.Deploy.Pixie -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Deploy.Pixie.exe
  Cosmos.Build.Common -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Build.Common.dll
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.DebugConnectors\DebugConnectorEdison.cs(216,17): warning CS0162: Unreachable code detected [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.DebugConnectors\Cosmos.Debug.DebugConnectors.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.DebugConnectors\DebugConnectorEdison.cs(143,22): warning CS0414: The field 'DebugConnectorEdison.mKernelSent' is assigned but its value is never used [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.DebugConnectors\Cosmos.Debug.DebugConnectors.csproj]
  Cosmos.Debug.Common -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Debug.Common.dll
  XSharp -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\XSharp.dll
  Cosmos.Debug.DebugConnectors -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Debug.DebugConnectors.dll
  Determining projects to restore...
  All projects are up-to-date for restore.
  Cosmos.Debug.GDB -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Debug.GDB.exe
  Cosmos.Debug.Kernel -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Debug.Kernel.dll
  Cosmos.Debug.HyperVServer -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Debug.HyperVServer.exe
  Cosmos.Debug.HyperVServer -> C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Debug.Hosts\obj\Debug\netstandard2.0\HyperVServer\
  Cosmos.Debug.Hosts -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Debug.Hosts.dll
  Cosmos.Common -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Common.dll
C:\Program Files\dotnet\sdk\5.0.101\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [C:\Users\illus\Documents\Cosmos-master\source\TheRingMaster\TheRingMaster.csproj]
C:\Program Files\dotnet\sdk\5.0.101\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [C:\Users\illus\Documents\Cosmos-master\Tools\NASM\NASM.csproj]
  Cosmos.Debug.HyperVServer -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Debug.HyperVServer.exe
  Cosmos.Deploy.USB -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Deploy.USB.exe
  VSPropertyPages -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\VSPropertyPages.dll
  NASM -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\NASM.dll
  Cosmos.TestRunner.TestController -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.TestRunner.TestController.dll
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.ProjectSystem\ProjectSystem\VS\PlugsProjectTreeProvider.cs(41,44): warning CS0618: 'ProjectTreeProviderBase.TreeUpdateResult.TreeUpdateResult(IProjectTree, bool, IImmutableDictionary<NamedIdentity, IComparable>)' is obsolete: 'LazyFill parameter is no longer supported.' [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.ProjectSystem\Cosmos.VS.ProjectSystem_wkqeviam_wpftmp.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.ProjectSystem\ProjectSystem\VS\PlugsProjectTreeProvider.cs(88,44): warning CS0618: 'ProjectTreeProviderBase.TreeUpdateResult.TreeUpdateResult(IProjectTree, bool, IImmutableDictionary<NamedIdentity, IComparable>)' is obsolete: 'LazyFill parameter is no longer supported.' [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.ProjectSystem\Cosmos.VS.ProjectSystem_wkqeviam_wpftmp.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.ProjectSystem\ProjectSystem\VS\PropertyPages\OldPropertyManager.cs(25,35): warning CS0067: The event 'OldPropertyManager.ConfigurationsChanged' is never used [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.ProjectSystem\Cosmos.VS.ProjectSystem_wkqeviam_wpftmp.csproj]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(3075,5): error MSB3821: Couldn't process file ProjectSystem\VS\PropertyPages\OldCosmosPropertyPageControl.resx due to its being in the Internet or Restricted zone or having the mark of the web on the file. Remove the mark of the web if you want to process these files. [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.ProjectSystem\Cosmos.VS.ProjectSystem.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\ToolWindows\StackUC.xaml.cs(59,13): warning VSTHRD001: Await JoinableTaskFactory.SwitchToMainThreadAsync() to switch to the UI thread instead of APIs that can deadlock or require specifying a priority. [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\Cosmos.VS.Windows.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\ToolWindows\StackUC.xaml.cs(92,13): warning VSTHRD001: Await JoinableTaskFactory.SwitchToMainThreadAsync() to switch to the UI thread instead of APIs that can deadlock or require specifying a priority. [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\Cosmos.VS.Windows.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\ToolWindows\RegistersControl.xaml.cs(20,13): warning VSTHRD001: Await JoinableTaskFactory.SwitchToMainThreadAsync() to switch to the UI thread instead of APIs that can deadlock or require specifying a priority. [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\Cosmos.VS.Windows.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\CosmosWindowsPackage.cs(67,17): warning VSSDK006: Check whether the result of GetService calls is null. [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\Cosmos.VS.Windows.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\ToolWindows\AssemblyUC.xaml.cs(605,13): warning VSTHRD001: Await JoinableTaskFactory.SwitchToMainThreadAsync() to switch to the UI thread instead of APIs that can deadlock or require specifying a priority. [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.VS.Windows\Cosmos.VS.Windows.csproj]
  Cosmos.VS.Windows -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.VS.Windows.dll
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2123,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "DapperExtensions.StrongName". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [C:\Users\illus\Documents\IL2CPU\source\IL2CPU.Debug.Symbols\IL2CPU.Debug.Symbols.csproj]
C:\Users\illus\Documents\IL2CPU\source\IL2CPU.Debug.Symbols\DebugInfo.cs(2,7): error CS0246: The type or namespace name 'DapperExtensions' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\illus\Documents\IL2CPU\source\IL2CPU.Debug.Symbols\IL2CPU.Debug.Symbols.csproj]
C:\Users\illus\Documents\IL2CPU\source\IL2CPU.Debug.Symbols\DebugInfo.cs(3,7): error CS0246: The type or namespace name 'DapperExtensions' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\illus\Documents\IL2CPU\source\IL2CPU.Debug.Symbols\IL2CPU.Debug.Symbols.csproj]
C:\Users\illus\Documents\IL2CPU\source\IL2CPU.Debug.Symbols\DebugInfo.cs(4,7): error CS0246: The type or namespace name 'DapperExtensions' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\illus\Documents\IL2CPU\source\IL2CPU.Debug.Symbols\IL2CPU.Debug.Symbols.csproj]
  IL2CPU.API -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\IL2CPU.API.dll
  Cosmos.Core.DebugStub -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Core.DebugStub.dll
  Cosmos.Debug.Kernel.Plugs.Asm -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Debug.Kernel.Plugs.Asm.dll
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\obj\Debug\netstandard2.0\Cosmos.Core.AssemblyInfo.cs(25,12): warning CS1700: Assembly reference 'Cosmos.Core.Tests, PublicKey=Cosmos' is invalid and cannot be resolved [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\Cosmos.Core.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\obj\Debug\netstandard2.0\Cosmos.Core.AssemblyInfo.cs(26,12): warning CS1700: Assembly reference 'Cosmos.Core.Memory.Test, PublicKey=Cosmos' is invalid and cannot be resolved [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\Cosmos.Core.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\INTs.cs(235,29): warning CS0649: Field 'INTs.mLastKnownAddress' is never assigned to, and will always have its default value 0 [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\Cosmos.Core.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\ACPI.cs(89,30): warning CS0414: The field 'ACPI.SCI_EN' is assigned but its value is never used [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Core\Cosmos.Core.csproj]
  Cosmos.Core -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.Core.dll
  TheRingMaster -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\TheRingMaster.dll
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\PCIDeviceBridge.cs(13,21): warning CS0108: 'PCIDeviceBridge.SecondaryBusNumber' hides inherited member 'PCIDevice.SecondaryBusNumber'. Use the new keyword if hiding was intended. [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\Cosmos.HAL2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\BlockDevice\Ports\Sata.cs(112,21): warning CS0162: Unreachable code detected [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\Cosmos.HAL2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\BlockDevice\Ports\Satapi.cs(105,21): warning CS0162: Unreachable code detected [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\Cosmos.HAL2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\BlockDevice\Ports\Sata.cs(206,21): warning CS0162: Unreachable code detected [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\Cosmos.HAL2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\BlockDevice\Ports\Sata.cs(280,21): warning CS0162: Unreachable code detected [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\Cosmos.HAL2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\Drivers\Video\VGADriver.cs(828,20): warning CS0219: The variable 'values' is assigned but its value is never used [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.HAL2\Cosmos.HAL2.csproj]
  Cosmos.HAL2 -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.HAL2.dll
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\PCSpeaker.cs(37,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://en.wikipedia.org/wiki/Duration_(music)' [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\Cosmos.System2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\PCSpeaker.cs(52,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://en.wikipedia.org/wiki/Musical_note' [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\Cosmos.System2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\PCSpeaker.cs(53,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://en.wikipedia.org/wiki/Audio_frequency' [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\Cosmos.System2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\obj\Debug\netstandard2.0\Cosmos.System2.AssemblyInfo.cs(24,12): warning CS1700: Assembly reference 'Cosmos.System.Tests, PublicKey=Cosmos' is invalid and cannot be resolved [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\Cosmos.System2.csproj]
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\Graphics\VBECanvas.cs(84,9): warning CS1587: XML comment is not placed on a valid language element [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2\Cosmos.System2.csproj]
  Cosmos.System2 -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.System2.dll
C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2_Plugs\System\MathImpl.cs(391,26): warning CS0675: Bitwise-or operator used on a sign-extended operand; consider casting to a smaller unsigned type first [C:\Users\illus\Documents\Cosmos-master\source\Cosmos.System2_Plugs\Cosmos.System2_Plugs.csproj]
  Cosmos.System2_Plugs -> C:\Users\illus\Documents\Cosmos-master\Build\VSIP\Cosmos.System2_Plugs.dll

========================================
    Error
========================================

System.Exception: The process failed to execute!
   at Cosmos.Build.Builder.BuildTasks.ProcessBuildTaskBase.WaitForExit(Process process) in C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Build.Builder\BuildTasks\ProcessBuildTaskBase.cs:line 67
   at Cosmos.Build.Builder.BuildTasks.ProcessBuildTaskBase.<>c__DisplayClass5_0.<RunAsync>b__2() in C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Build.Builder\BuildTasks\ProcessBuildTaskBase.cs:line 48
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable.ConfiguredTaskAwaiter.GetResult()
   at Cosmos.Build.Builder.ViewModels.MainWindowViewModel.<BuildAsync>d__27.MoveNext() in C:\Users\illus\Documents\Cosmos-master\source\Cosmos.Build.Builder\ViewModels\MainWindowViewModel.cs:line 128

**Steup (please complete the following information):**
Devkit downloaded from the master yesterday
 - Are you using an admin account?
 - yes
 - VS Version if applicable
2019

**Additional context**

**Comment by valentinbreiz** [comment]
Did you install InnoSetup?

**Comment by quajak** [comment]
Seems same problem as https://github.com/CosmosOS/Cosmos/issues/1100

**Comment by illus-coder** [comment]
innosetup installed before the error occurred, and for good measure I went and unblocked all resx files but I am still getting the error.
Pastebin: https://pastebin.com/Lit1zT4X

**Comment by quajak** [comment]
Duplicate of https://github.com/CosmosOS/Cosmos/issues/1417#issuecomment-661348510 Rename Cosmos-master to Cosmos.
